home_audit.node.js

```diff
@@ -722,15 +711,10 @@ let validationRules = {
     _window_area: function(value, isFront) {
         let wall_area = this._get_wall_area();
         if (wall_area) {
-            //Windows have API max area of 999
-            //TODO: Clarify that this really should be the maximum (documentation has less than calculated wall area)
-            if (wall_area > 999) {
-                wall_area = 999;
-            }
-            return new Validation(TypeRules._float(value, 0, wall_area), BLOCKER);
-        } else {
-            return new Validation("Must enter Conditioned floor area, Interior floor-to-ceiling height, and stories above ground level", ERROR);
+            return new Validation(TypeRules._float(value, 0, wall_area), ERROR);
         }
+        //Windows have API max area of 999
+        return new Validation(TypeRules._float(value, 0, 999), BLOCKER);
```

Comment:
Is it possible to construct a building that has more than 999 sqft of wall area? I'm wondering if we don't need logic that guarantees we'll return the BLOCKER in such a case.

Reply:
Sorry I was unclear. I'll try an example:

I create a HUGE home that has 2000sqft wall area. I enter 1200sqft of windows. When this validation rule runs, `if (wall_area)` will evaluate to `TRUE` and the return value will be the `_float()` rule that checks for a valid float in the range 0-2000. But then the value submitted will trigger a SOAP error, because 1200 is greater than the 999 maximum allowed for the field by the SOAP API.

I suggest that, if `value` is greater than 999, we should always return the `BLOCKER` rule, and otherwise the `ERROR` rule.